cmd/migrate/db/mariadb.go

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+package db
```

Comment:
```suggestion
package mariadb
```
Suggestion to rename this file to 
`cmd/migrate/mariadb/conn.go` :thinking: ?